editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -1200,6 +1200,7 @@ describe('conditionals in the navigator', () => {
       ),
     ).toEqual(`  regular-utopia-storyboard-uid/scene-aaa
     regular-utopia-storyboard-uid/scene-aaa/containing-div
+      regular-utopia-storyboard-uid/scene-aaa/containing-div/then-then-div
       regular-utopia-storyboard-uid/scene-aaa/containing-div/conditional1
```

Comment:
this change is reflecting a real bugfix: on master we wrongly put the conditionals before their siblings in the navigator